bench/Manifest.toml

```diff
@@ -2,9 +2,9 @@
 
 [[ADI]]
 deps = ["Dierckx", "Distributions", "FillArrays", "HCIToolbox", "ImageTransformations", "LinearAlgebra", "NMF", "Parameters", "Photometry", "ProgressLogging", "Reexport", "Rotations", "Setfield", "StaticArrays", "StaticKernels", "Statistics", "StatsBase", "TSVD", "UnPack"]
-path = "/Users/miles/dev/julia/ADI"
+git-tree-sha1 = "6bb9d25e57347c9dbd4c9eca42f019179e019cc8"
 uuid = "904a6c7d-4c1b-562f-9573-ab2e7e1c7946"
-version = "0.5.0"
+version = "0.5.2"
```

Comment:
As an alternative we can `add ADI#master`.  The important is to not have a local path in the manifest.